draft-schlesinger-cfrg-act.md

```diff
@@ -825,11 +840,61 @@ interoperability. The curve specification is included to prevent cross-curve
 attacks and ensure implementations using different curves cannot accidentally
 interact.
 
+### Sigma Proofs and Fiat-Shamir Transform {#sigma-and-fs}
+
+Proofs of knowledge are based on interactive sigma protocols, which
+are made non-interactive through the Fiat-Shamir transform {{FST}}.
+The concrete construction uses two interfaces:
+one for describing the sigma protocol, and
+another for generating non-interactive zero-knowledge proofs.
+
+A proof of knowledge is constructed for the relation
+$$R = {(x, w) : x \in L, w \in W(x)}$$
+where the language (`L`) is the set of linear relations between the
+statement (`x`) and the witness (`w`) in the set (`W(x)`) of valid
+witnesses for `x`.
+The statement is expressed as linear combinations of scalars and group
+elements, while the witness is represented as a list of scalars.
+The LinearRelation interface, described in {{!I-D.irtf-cfrg-sigma-protocols}}, allows
+constructing a sigma protocol for the relation above.
```

Comment:
Relations should be expressed mathematically in the same way as the sigma protocols draft itself. that is, this should be aligned more with https://mmaker.github.io/draft-irtf-cfrg-sigma-protocols/draft-irtf-cfrg-sigma-protocols.html#section-2-4